todo_app/flask_config.py

```diff
@@ -1,9 +1,21 @@
 import os
 
-
 class Config:
     def __init__(self):
         """Base configuration variables."""
         self.SECRET_KEY = os.environ.get('SECRET_KEY')
```

Comment:
minor, once you've removed `session_items.py` you should be able to remove this env var too